aws-auth-cognito/README.md

```diff
@@ -0,0 +1,145 @@
+## User Acceptance Testing Guide
```

Comment:
Maybe yank this before merging? This was super helpful and thank you!

Reply:
Done